addons/addon-base-raas/packages/base-raas-services/lib/data-source/data-source-reachability-service.js

```diff
@@ -110,20 +111,24 @@ class DataSourceReachabilityService extends Service {
         },
       );
     }
+
     if (!_.isEmpty(unreachableAppRoles)) {
       statusMsg = `ERR|||Error getting information from ${unreachableAppRoles.length} application roles. 
       Please update the cloudformation template on data source account ${id}`;
     }
-    await accountService.updateStatus(requestContext, dataSourceAccount, { status: newStatus, statusMsg });
-    const outputVal = { status: newStatus, statusMsg };
+
+    const entity = await accountService.updateStatus(requestContext, dataSourceAccount, {
+      status: newStatus,
+      statusMsg,
+    });
 
     // Write audit event
     await this.audit(requestContext, {
       action: 'check-dsAccount-reachability',
       body: { id, type },
     });
 
-    return outputVal;
+    return entity;
```

Comment:
Lets also update the expected response (from status to entity object) in the Quip doc.